modules/world/world.hpp

```diff
@@ -159,9 +167,14 @@ class World : public commons::BaseType {
     ClearObjects();
     ClearEvaluators();
   }
+  void AddLabels(const LabelFunctions& label_evaluators);
```

Comment:
Labels should be contained in the evaluator.
If these are required for the LTL rule evaluator, consider to move them there.